exporter/trace/trace.go

```diff
@@ -29,11 +34,19 @@ import (
 type traceExporter struct {
 	o         *options
 	projectID string
+	bundler   *bundler.Bundler
```

Comment:
@rghetia As far as I remember, there's some issues on using bundler and that's the reason why I avoided using bundler and used BatchWriteSpans in the first implementation. Could you share the background for the record?

Reply:
@ymotongpoo I kind of remember some issue with bundler but I don't remember the details.

Reply:
For record: at least the report in OC pointed the possibility of memory leak.
https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/issues/71

Reply:
From that issue, looks like the memory leak issue only appears when the backend is not reachable. Should be relatively easy to reproduce the issue?

Reply:
@ymotongpoo I don't really understand that issue in the ocagent repo. It says 'as long as there is a valid connection to the collector, the “handler” function, i.e. uploadTraces, will be called on the bundle to offload the bundled traces to the collector.' Isn't the handler function called regardless of whether there is a connection? That connection check in the issue is already inside the `uploadTraces` function. And in this PR, there is no such check for a connection, the upload function just calls `BatchWriteSpans`.
Also, since the bundler's [BufferedByteLimit](https://github.com/googleapis/google-api-go-client/blob/master/support/bundler/bundler.go#L76-L78) is set [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/39/files#diff-548b706e759db54d6bb8ae08370364caR86), the bundler will not keep more than that many bytes in memory.